packages/lesswrong/components/ea-forum/EAHomeCommunityPosts.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { Components, registerComponent } from '../../lib/vulcan-lib';
+import { Link } from '../../lib/reactRouterWrapper';
+import { AnalyticsContext } from '../../lib/analyticsEvents';
+import moment from '../../lib/moment-timezone';
+import { useTimezone } from '../common/withTimezone';
+
+const styles = (theme: ThemeType): JssStyles => ({
+
+})
+
+const EAHomeCommunityPosts = ({classes}:{classes: ClassesType}) => {
+  const { timezone } = useTimezone()
+  const { SingleColumnSection, PostsList2, SectionTitle, SectionFooter } = Components
+
+  const now = moment().tz(timezone)
+  const dateCutoff = now.subtract(90, 'days').format("YYYY-MM-DD")
+
+  const recentPostsTerms = {
+    view: "magic",
+    filterSettings: {tags: [{
+      tagId: 'ZCihBFp5P64JCvQY6', // "Community" topic
```

Comment:
This is pulled out in https://github.com/ForumMagnum/ForumMagnum/pull/6617 so I'll replace it in that PR